cpp/src/jit/cache.cpp

```diff
@@ -112,9 +123,27 @@ jitify2::ProgramCache<>& get_program_cache(jitify2::PreprocessedProgramData prep
   auto existing_cache = caches.find(preprog.name());
 
   if (existing_cache == caches.end()) {
-    auto res = caches.insert(
-      {preprog.name(),
-       std::make_unique<jitify2::ProgramCache<>>(100, preprog, nullptr, get_program_cache_dir())});
+    size_t kernel_limit_proc = std::numeric_limits<size_t>::max();
+    size_t kernel_limit_disk = std::numeric_limits<size_t>::max();
+
```

Comment:
```suggestion
```

Reply:
Also, use `std::size_t` instead of `size_t`.